artifactory/commands/transferfiles/state_test.go

```diff
@@ -129,3 +122,38 @@ func getRepoFromState(t *testing.T, repoKey string) *Repository {
 	assert.NoError(t, err)
 	return repo
 }
+
+func setupStateTest(t *testing.T) (cleanUp func()) {
```

Comment:
Our convention is to call this function init:
```suggestion
func initStateTest(t *testing.T) (cleanUp func()) {
```